src/linter/yaml/UI5YamlLinter.ts

```diff
@@ -0,0 +1,87 @@
+import {LintMessageSeverity} from "../LinterContext.js";
```

Comment:
We try to follow the Camel Case rules outlined in Google's style guide: https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined

Especially this part is relevant here:
> _[...] lowercase everything **(including acronyms)**, then uppercase only the first character [...]_

Based on that I would rename this module to `Ui5YamlLinter` or maybe just `YamlLinter` to be aligned with the directory name?

Same with some of the variables and function names 🙂